<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardWindow</class>
 <widget class="QWidget" name="BoardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="boardView">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>771</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExitCurrentGame">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Exit Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRollDice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Roll dice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEndTurn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>End turn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUndo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView {
	border: 1px black;
}</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="historyboardView">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>771</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="lbBlack">
      <property name="text">
       <string>Black</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbWhite">
      <property name="text">
       <string>White</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lbScoreWhite">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lbScoreBlack">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
